### PR TITLE
Update docs to include simplified process of adding Spaces to ArXiv

### PR DESCRIPTION
**Pull request by abidlabs**
As discussed with @julien-c @osanseviero, here's an addition to the docs to show the simplified process of adding Spaces to ArXiv Labs

### CODE REVIEW

File: docs/hub/spaces-add-to-arxiv.md [hunk 1]
```diff
@@ -8,15 +8,22 @@ Thanks to this integration, users can now find the most popular demos for a pape
...
+We'll cover two different ways to add your Space to ArXiv and have it show up in the Demos tab. 
 
 **Prerequisites**
 
 * There's an existing paper on ArXiv that you'd like to create a demo for
-* The paper is **associated with a model** that is on the Hugging Face Hub (or can be uploaded there)
 * You have built or (can build) a demo for the model on Spaces
 
-**Steps to Add Your Space to ArXiv**
+**Method 1 (Recommended): Linking from the Space Readme**
```
Comment (osanseviero) [inline docs/hub/spaces-add-to-arxiv.md:18]:
```suggestion
**Method 1 (Recommended): Linking from the Space README**
```

File: docs/hub/spaces-add-to-arxiv.md [hunk 2]
```diff
@@ -8,15 +8,22 @@ Thanks to this integration, users can now find the most popular demos for a pape
...
+
+The simplest way to add a Space to an ArXiv paper to include the link to the paper in the Space readme (README.md). It's good practice to include a full citation as well. You can see an example of a link and a citation on this [Echocardiogram Segmentation Space README](https://huggingface.co/spaces/abidlabs/echocardiogram-arxiv/blob/main/README.md).
```
Comment (osanseviero) [inline docs/hub/spaces-add-to-arxiv.md:20]:
```suggestion
The simplest way to add a Space to an ArXiv paper is to include the link to the paper in the Space README file (`README.md`). It's good practice to include a full citation as well. You can see an example of a link and a citation on this [Echocardiogram Segmentation Space README](https://huggingface.co/spaces/abidlabs/echocardiogram-arxiv/blob/main/README.md). 
```

**Review by osanseviero — APPROVED** [review]
Thanks!

**Comment by abidlabs** [comment]
Thanks for the review and suggestions @osanseviero!

**Pull request merged**